tests/end-to-end/backup-restore-test/backup-restore.md

```diff
@@ -0,0 +1,13 @@
+# Backup and Restore E2E Tests
+
+## Overview
+
+This project contains the end to end tests that it runs as part of the Kyma on Google Cloud Platform. The tests is written in Go. 
+
```

Comment:
Maybe we can add an introductory sentence here, like:
```
The end-to-end test scenario looks as follows:

```

Reply:
I updated and changed the documentation a bit. Could you take a look again?